x-pack/plugins/gis/public/actions/store_actions.js

```diff
@@ -242,8 +243,8 @@ export function setMouseCoordinates({ lat, lon }) {
 
   return {
     type: SET_MOUSE_COORDINATES,
-    lat,
-    lon: safeLon,
+    lat: _.round(lat, DECIMAL_DEGREES_PRECISION),
+    lon: _.round(safeLon, DECIMAL_DEGREES_PRECISION),
```

Comment:
Should we do the rounding just at display-time instead? Are there any advantages of rounding the user at input-time (?)